src/Tizen.NUI/src/public/Utility/Palette.cs

```diff
@@ -127,7 +130,6 @@ private Palette(List<Swatch> swatcheList)
         /// <param name="pixelBuffer">A Target image's pixelBuffer.</param>
         /// <param name="paletteGeneratedEventHandler">A method will be called with the palette when generated.</param>
         /// <exception cref="ArgumentNullException">Thrown when the argument pixelBuffer, PaletteGeneratedEventHandler is null.</exception>
-        [EditorBrowsable(EditorBrowsableState.Never)]
         public static void GenerateAsync(PixelBuffer pixelBuffer, PaletteGeneratedEventHandler paletteGeneratedEventHandler)
```

Comment:
please use C# features for asynchronous methods. An asynchronous method should return [Task<T>](https://docs.microsoft.com/en-gb/dotnet/api/system.threading.tasks.task-1?view=netcore-3.0) or Task. Optionally be marked with `async` keyword - but you probably will not need that. This way user will call:
```
   var pallete = await Pallete.GenerateAsync(buffer);
```

I've noticed that you've already implemented a Task returning method, so just return that this task and don't expose the callback.